src/routes/events/FeaturedEvent.svelte

```diff
@@ -0,0 +1,47 @@
+<script lang="ts">
+    import Button from '$lib/components/Button.svelte';
+
+    export let tags: string[] = [];
+    export let schedule: string[] = [];
+</script>
+
+<!-- Featured Card Event
+tags and schedule - pass as props
+image, event name, description - pass as slot -->
```

Comment:
Looks like you were trying to do an [`@component`](https://svelte.dev/docs/faq#how-do-i-document-my-components) doc-comment here.

Reply:
Ah should clarify that I didn't modify the original component from #61 much besides adding dark mode styles. My main goal for this PR was primarily to add the existing component to the page. Still, will go over the requested changes as soon as I can.